lib/matplotlib/tri/triangulation.py

```diff
@@ -80,18 +80,15 @@ def __init__(self, x, y, triangles=None, mask=None):
                     f'range 0 <= i < {len(self.x)} but found value '
                     f'{self.triangles.min()}')
 
-        if mask is not None:
-            self.mask = np.asarray(mask, dtype=bool)
-            if self.mask.shape != (self.triangles.shape[0],):
-                raise ValueError('mask array must have same length as '
-                                 'triangles array')
-
         # Underlying C++ object is not created until first needed.
         self._cpp_triangulation = None
 
         # Default TriFinder not created until needed.
         self._trifinder = None
 
+        # Set mask
```

Comment:
```suggestion
```
The code already says this :).

Reply:
I'll sort that out, but not trigger a new CI  just yet.